standalone/src/main/java/io/pravega/local/InProcPravegaCluster.java

```diff
@@ -127,39 +127,75 @@
     private String keyPasswordFile;
     private String jksKeyFile;
 
-    public static final class InProcPravegaClusterBuilder {
-        public InProcPravegaCluster build() {
-            //Check for valid combinations of flags
-            //For ZK
-            Preconditions.checkState(isInProcZK || zkUrl != null, "ZkUrl must be specified");
-
-            //For controller
-            Preconditions.checkState(isInProcController || controllerURI != null,
```

Comment:
fields with a Default value now works differently and you do not have a field that holds the value